### DISCUSSION TITLE
🙋Teruglevering EN verbruik op hetzelfde moment? (Landis E350 SMR 4.2 model)

### DISCUSSION THREAD

**Discussion by simkin** [dsmrreader/dsmr-reader#1662] [Questions & answers / How-to's] [answered]
### Description

Ik heb het idee dat mijn slimme meter van het type Landis E350 SMR 4.2 model soms onterecht verbruik registreert terwijl er meer dan genoeg teruglever capaciteit is (van de zonepanelen). Dit lijkt een bekend probleem zijn maar wil graag achterhalen of ik de dsmr-reader metingen als bewijs kan aanleveren of niet. Mogelijk dat er meerdere telegram lezingen worden gebundeld die dit gedrag logisch kunnen verklaren? 

Mocht dat het geval zijn vraag ik me af of het mogelijk is om een systeem-notificatie toe te voegen die de gebruiker informeert dat er een vreemde telegram is gelezen die verbruik registreert, waar dat niet het geval mag zijn vanwege de hoge opbrengst.

Hieronder een voorbeeld van 'dubbele' metingen in de grafieken:
![image](https://user-images.githubusercontent.com/958263/177766798-ffd3f65e-71a5-4fed-9061-435ae26815e1.png)
(Het zou mooi zijn dat als ik op meetpunt balletje (e.g. 18:19) click, desbetreffende telegram, die als basis vormt van de grafiek data, getoond wordt)

Mijn probleem is oa ook op het tweakers.net forum beschreven: https://gathering.tweakers.net/forum/list_messages/2126210

### DSMR-reader version

DSMR-reader v5.3

### DSMR-reader platform

Docker (e.g. Xirixiz)

**Comment by dennissiemensma** [comment, marked as answer]
Bedankt voor je verzoek. Ik denk dat je relatief makkelijk de informatie kan vinden die je zoekt.

- DSMR-reader **groepeert** standaard metingen per minuut. Je kunt beginnen door die optie **uit te zetten** in: `admin/dsmr_consumption/consumptionsettings/1/change/`

----

- DSMR-reader **bewaart geen telegrammen**, alleen de laatste. Je kunt de laatste inzien op: `/admin/dsmr_datalogger/meterstatistics/`
- Je kunt echter ook gewoon DEBUG logging aanzetten voor een tijdje. Dan **logt** die ook alle telegrammen in de logfiles.
- https://dsmr-reader.readthedocs.io/nl/v5/how-to/troubleshooting/logfiles.html
- https://dsmr-reader.readthedocs.io/nl/v5/how-to/troubleshooting/enabling-debug-logging.html

----

- Overigens kan het altijd zijn dat je teruglevering en verbruik tegelijkertijd hebt. Sterker nog, ik kwam daar pas vorig jaar achter via #1324

- Hoe zie je grafiek er uit op fase-niveau? Want stel dat je hetzelfde hebt als deze in dat topic, dan is het verklaarbaar: https://gathering.tweakers.net/forum/list_message/71339424#71339424

- Ik ben **geen** elektra-expert, dus ik weet ook niet hoe elektriciteit werkt in de zin van terugduwen in je huishouden. Maar kan het zijn dat je welliswaar genoeg "verbruikt" in je huis, maar dat desondanks toch een deel van je opwekking via de meter sijpelt en teruggeleverd wordt aan het net?

**Reply by dennissiemensma** [reply]
@simkin heb je de informatie gevonden die je zocht? Of wellicht juist niet zocht?

**Reply by simkin** [reply]
Ik kijk het even aan met de groepering setting op uit. Als ik dan nog steeds raar gedrag zie duik ik er wat verder in. Ideaal zou een notificatie zijn maar ik snap dat dit zo weinig voor komt dat het haast niet de moeite is om toe te voegen. Bij deze het issue gesloten. Bedankt voor je hulp!
